neurom/fst/tests/test_get_features.py

```diff
@@ -321,39 +321,39 @@ def test_total_length_pop():
     feat = 'total_length'
 
     nt.ok_(np.allclose(fst.get(feat, POP),
-                       [840.47744821, 418.83424432, 13250.82577394]))
+                       [840.68522362, 418.83424432, 13250.82577394]))
```

Comment:
I see a discrepancy in the Z coordinate of the second point in each section between the old and new h5 files. This could be an artefact of the conversion. I will look into it.